hack/image-stream-build-config.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
```

Comment:
Can we rename this to something that ends with '-template.yaml'  to better identify its a resource that will require additional processing inorder to properly use.